src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -95,12 +114,92 @@ public void onCallback(final JSONObject options) {
             }
         }).timingOptions(timingCategory, timingVariableName, 0);
     }
+    
+    private void init() {
```

Comment:
The remaining of the class seems to use tabs instead of spaces. Alignment is wrong